appcli/models/cli_context.py

```diff
@@ -217,3 +235,34 @@ def get_variables_manager(self) -> VariablesManager:
             key_file=self.get_key_file(),
             application_context_files_dir=self.application_context_files_dir,
         )
+
+    def get_helm_dir(self) -> Path:
+        """Parent directory for all helm related configs.
```

Comment:
I don't love the fact that these things are in the root CLI context. They make no sense when not using the Helm orchestrator.

We should move the three of them into the `HelmOrchestrator` class itself. Just like how the `DockerComposeOrchestrator` handles its own specific paths (`docker_compose_file`, `docker_compose_override_directory`, etc).